Drv/LinuxUartDriver/LinuxUartDriver.cpp

```diff
@@ -309,11 +309,7 @@ void LinuxUartDriver ::serialReadTaskEntry(void* ptr) {
     FW_ASSERT(ptr != nullptr);
     Drv::RecvStatus status = RecvStatus::RECV_ERROR;  // added by m.chase 03.06.2017
     LinuxUartDriver* comp = reinterpret_cast<LinuxUartDriver*>(ptr);
-
     while (!comp->m_quitReadThread) {
-        // wait for data
-        int sizeRead = 0;
-
         Fw::Buffer buff = comp->allocate_out(0, Drv::UART_READ_ALLOCATION_REQUEST_SIZE);
```

Comment:
I wish I had noticed this earlier. I think this should be a parameter to the constructor, since a global config may not work if allocation size is tuned by driver instance.

Reply:
It is fixed now.